.eslintignore

```diff
@@ -1 +0,0 @@
-node_modules/*
```

Comment:
I found the line: http://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories

"In addition to any patterns in a .eslintignore file, ESLint always ignores files in node_modules/**."